addons/sourcemod/scripting/zr_personal_skins.sp

```diff
@@ -1,27 +1,26 @@
 #include <sourcemod>
 #include <sdktools>
-#include <clientprefs>
 #include <smlib>
 #include <multicolors>
 #include <zombiereloaded>
+#include <utilshelper>
+
+#undef REQUIRE_PLUGIN
+#tryinclude <vip_core>
+#define REQUIRE_PLUGIN
 
 #pragma semicolon 1
 #pragma newdecls required
 
-#define CHANGE_MODE_NOT_CHANGED_YET 0
-#define CHANGE_MODE_CHANGED 1
-#define CHANGE_MODE_CHANGESKIN_TO_PSKIN 2
-#define CHANGE_MODE_CHANGESKIN_TO_ACTUAL 3
-
-int 
-		g_iClientChangeMode[MAXPLAYERS + 1] = { CHANGE_MODE_NOT_CHANGED_YET, ... },
-		g_iClientPreviousChangeMode[MAXPLAYERS + 1] = { CHANGE_MODE_NOT_CHANGED_YET, ... };
+//#define DEBUG
+#define Grp_Zombie 		"Personal-Skin-Zombie"
```

Comment:
we use sm_flags for zombiereloaded and we would need to enable zr_permissions_use_groups so maybe just add people with personal skins to a specific sourcemod flag:
- https://wiki.alliedmods.net/Adding_Admins_(SourceMod)#Levels

Reply:
`zr_permissions_use_groups` (CVAR_PERMISSIONS_USE_GROUPS) is only used for Admins commands.
The current code doesn't need this cvar to be on 1.

Reply:
yes but its really confusing when you look at playerclasses since there is a mix of behavior -> most classes use sm_flags and only personal skins uses groups

Reply:
but i guess it can be acceptable so in this case put cvars instead of defines